rocketpy/rocket/parachute.py

```diff
@@ -238,3 +212,11 @@ def all_info(self):
         """Prints all information about the Parachute class."""
         self.info()
         # self.plots.all() # Parachutes still doesn't have plots
+
+        # .. note::
+
+        #     The drag force exerted on the parachute is computed by \
+        #     the equation F = ((1/2)*rho*V^2)*cd_s, where rho is the \
+        #     freestream air density, ``V`` is the relative velocity between \
+        #     the rocket and the air, and ``cd_s`` is the drag coefficient \
+        #     times the reference area of the parachute.
```

Comment:
It seems a bit odd to have these lines at the end of the class in the info method. Is this the intended location?

Reply:
This was certainly not made on purpose.